bin/test_integration_rhel

```diff
@@ -1,23 +1,57 @@
 #!/bin/bash -ex
 
+function print_help() {
+  cat << EOF
+
+Run integration tests on RHEL container
+
+Usage: test_integration_rhel [options]
+
+    -h, --help            Shows this help message.
+
+    --rhel-version=7  Run in RHEL 7 container.
+
+    --rhel-version=8  Run in RHEL 8 container.
+
+    -d, --debug           Step into the container.
+
+EOF
+}
+
+while true ; do
+  case "$1" in
+    #--rhel-version=* ) RHEL_VERSION="${option#*=}" ; shift ;;
+    --rhel-version=7 ) RHEL_VERSION="7" ; shift ;;
+    --rhel-version=8 ) RHEL_VERSION="8" ; shift ;;
+    -d|--debug ) DEBUG="true" ; shift ;;
+    -h|--help ) print_help ; shift ;;
+     * ) if [ -z "$1" ]; then break; else echo "$1 is not a valid option"; exit 1; fi;;
+  esac
+done
+
 cleanup() {
   echo "Cleaning up..."
   echo "Cleaning up... logs in cleanup.log"
   docker-compose rm --stop --force
 }
 
-DEBUG="false"
```

Comment:
why did we remove this? this is important

Reply:
@eranha

Reply:
@sigalsax it is now written in line 24. or do we still need to explicitly set this to FALSE?